test/Module/CommandCommonOptionsTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2017-2018 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
Minor nitpick here; since this is a new file, we only need the start date, which should be this year (2019). I can fix that when I merge, however.

Reply:
Ah yes. Copy/paste error. Did not have a closer look at the header comment. Thanks.